integration-tests/steps/stacks.py

```diff
@@ -442,6 +473,16 @@ def step_impl(context, a_or_no, kind):
         assert getattr(writer, difference_property) is test_value
 
 
+@then('stack drift status is "{desired_status}"')
+def step_impl(context, desired_status):
+    assert context.output[0]["StackDriftStatus"] == desired_status
+
+
+@then('stack resource drift status is "{desired_status}"')
+def step_impl(context, desired_status):
+    assert context.output[0][1]["StackResourceDrifts"][0]["StackResourceDriftStatus"] == desired_status
```

Comment:
This should loop through every item of `context.output` and then every resource and check the status of each resource. Otherwise, you're just checking the first item of the first stack, which isn't really a great option. Also, it's not immediately clear why you're accessing the second item of `context.output[0]`. I remember that method is likely returning a tuple, but that's not clear here. I think it would be a _lot_ clearer if you did something like this:

```suggestion
    for drift_status, resource_drifts in context.output:
        for resource_drift in resource_drifts:
            assert resource_drift["StackResourceDriftStatus"] == desired_status
```